### ISSUE TITLE
Relational Databases Cert to do list

### ISSUE THREAD

**Issue by moT01** [freeCodeCamp/freeCodeCamp#42509]
| inProgress | Complete | |
| :--: | :--: | --- |
| ✓ | ✓ | Reorder superBlocks to the proper order - RDBMS Should be right after front end certs. Probably right after Data Viz for now - but the long term plan is to combine data viz and front end libraries |
| ✓ | ✓ | Find and add an icon for relational databases cert |
| ✓ | ✓ | Remove the vertical scroll bar that shows up from line-height added by bootstrap |
| ✓ | ✓ | Transfer all tutorials to fCC organization and update URL's in challenge md files |
| ✓ | | Finish creating the tutorials |
| | | Add tests if possible |
| ✓ | ✓ | Write intros for placeholders |
| | | Add cert back to settings page (bottom of cert-and-project-map file)|
| ✓ | | Add API endpoints for submitting tutorials/projects (https://github.com/freeCodeCamp/CodeAlly-CodeRoad-freeCodeCamp/issues/18) |
| | | Add "Are you sure you want to leave?" pop up to stop accidental closing of tutorials (https://github.com/freeCodeCamp/CodeAlly-CodeRoad-freeCodeCamp/issues/12) |
| | | Change graphql query to static query in Show.js? |
| | | Add block intro page since we aren't getting rid of the others?

If anyone wants to fix any of these remaining issues, feel free to make a PR. Also, some of them may require more discussion - that can take place here or on a separate issue if it's a more significant feature.

**Comment by Sembauke** [comment]
Is the Relational databases certification in show upcoming changes?

**Comment by moT01** [comment]
Yes. If you want to test out any of the tutorials, some feedback would be great 👍 Once you click a tutorial, log in to CodeAlly with your GitHub account and run the tutorial. After the VM loads, open VSCode's command palette (I think click `view` -> `Command Palette`) and run `CodeRoad: Start` - it should be the first option. From there, you should just be able to go through the tutorial. The advanced bash, github, and final boss projects aren't finished so I would go through one of the others. There's a couple in the group of others that may have some issues still, but they should be completable.

**Comment by Sembauke** [comment]
Hey Tom, 

I am trying to start the VM but something seems to go wrong when trying to log in.

https://gyazo.com/45fae26f11523117680e68fb790d6f11

**Comment by moT01** [comment]
I haven't seen that one before. There shouldn't be any permissions needed to run anything. I suppose other than just being logged in to [CodeAlly](https://codeally.io/). If you go there, are you logged in? You could try clearing all the site data and logging in through their site first (using the login with github method). Was there any errors in the console? I just logged out and logged back in with two different users and browsers, it started up for me.

**Comment by EugeneGohh** [comment]
Can I work on GraphQL query? Also, it seems like I can't find `Show.js`.

**Comment by moT01** [comment]
I'm not sure that one is necessary @EugeneGohh. I put a question mark at the end of it because I wasn't sure. I'm pretty sure I used to see a warning about it in the console, but I just ran the code and don't see it anymore - so I'm guessing that issue would be fine how it is if we want to leave it. But if you want to look into it to find out if it's better to use one over the other, the [show file is here](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/client/src/templates/Challenges/codeally/show.tsx). Here's a link to a [gatsby page on the two different types of queries](https://www.gatsbyjs.com/docs/static-vs-normal-queries/), as well.
